### ISSUE TITLE
Create accessPackageResourceRequest application type permission support

### ISSUE THREAD

**Issue by gizamo** [microsoftgraph/microsoft-graph-docs#15381]
Hi,

I would like to know more about planned improvements regarding application type permission support, for the Create accessPackageResourceRequest. Will this be available in the near future, for the graph API Beta version?
The lack of this functionality is a nuisance, as 4 other major steps within Identity Governance (create catalog, create access package, add resource to access package, and create policy), do support application type permissions.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: ec50888f-1fe5-5d73-3a4e-cfd698d6b455
* Version Independent ID: 4786d6a3-0108-20a2-d0fc-b1e727682d57
* Content: [Create accessPackageResourceRequest - Microsoft Graph beta](https://docs.microsoft.com/en-us/graph/api/entitlementmanagement-post-accesspackageresourcerequests?view=graph-rest-beta&tabs=http)
* Content Source: [api-reference/beta/api/entitlementmanagement-post-accesspackageresourcerequests.md](https://github.com/microsoftgraph/microsoft-graph-docs/blob/main/api-reference/beta/api/entitlementmanagement-post-accesspackageresourcerequests.md)
* Product: **governance**
* Technology: **microsoft-graph**
* GitHub Login: @markwahl-msft
* Microsoft Alias: **MSGraphDocsVteam**

**Comment by gizamo** [comment]
Is there any progress made regarding this? @markwahl-msft 
I'm looking forward to receive an update, or an indication of when application type permissions are going to be supported for this.

**Comment by markwahl-msft** [comment]
#15785

**Comment by gizamo** [comment]
@markwahl-msft 
Thank you for addressing this issue.
However, it seems that creating an access package resource using application type permission still does not work.
I receive a (400) bad request error, which is the same error I received when I tried to use an 'application type access token' before it was supported, after the pull request you linked.
This is not the error that should be obtained in the first place, as it is not a bad request, but rather a (401) unauthorized.
Only changing the 'application type access token' to the 'delegated type access token' allows again to successfully make the call. 
Hope this can be verified and fixed.

**Comment by gizamo** [comment]
@markwahl-msft 
The resourceType/originSystem are both 'AadGroup'.
The configured application permissions are EntitlementManagement.ReadWrite.All for both delegated type and application type. According to the documentation, this is the permission required to call that API.
Additionally, the application has Application.ReadWrite.All / Directory.Read.All / Group.Read.All , permissions of application type. But these are used for other purposes.

**Comment by gizamo** [comment]
@markwahl-msft 
I just tried my test again, to create access package resource using application type permission.
With additionally Group.ReadWrite.All application permissions granted to my application, I am able to create resources using application type! I'm not sure if this is intentional or not.

**Comment by markwahl-msft** [comment]
Yes, your application would need the permissions to modify groups to be able to add a group to an access package catalog.

**Comment by sergeyz7** [comment]
Any information on SharePoint sites please @markwahl-msft? The client app has got Site.FullControl.All - but comes back with 'The caller is not authorized to get sites.'
